app/controllers/catalog_controller.rb

```diff
@@ -175,6 +175,12 @@ def enforce_show_permissions(_opts = {})
     config.add_index_field solr_name("issued", :displayable),
                            label: "Issued Date"
 
+    config.add_index_field solr_name("description", :stored_searchable),
+                           label: "Summary",
+                           if: :collection?
```

Comment:
Descriptions are often way too long for non-Collections, especially in the case of ETDs.